staging/src/k8s.io/apiserver/pkg/storage/cacher/watch_cache_interval.go

```diff
@@ -114,6 +115,20 @@ func newCacheInterval(startIndex, endIndex int, indexer indexerFunc, indexValida
 	}
 }
 
+type sortableWatchCacheEvents []*watchCacheEvent
+
+func (s sortableWatchCacheEvents) Len() int {
+	return len(s)
+}
+
+func (s sortableWatchCacheEvents) Less(i, j int) bool {
+	return s[i].Key < s[j].Key
```

Comment:
is key = `namespace/name` ?

Reply:
it's the etcd key [that etcd is using to sort, as well as we're using elsewhere:
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/storage/cacher/watch_cache.go#L479-L491
]

Reply:
OK - I think I found it - https://github.com/kubernetes/kubernetes/blob/a05db0dd22a68a9c443a9f01cc1b8f6397fd6a9f/staging/src/k8s.io/apiserver/pkg/registry/generic/registry/store.go#L1556